### ISSUE TITLE
Fatal errors missing origin file and line

### ISSUE THREAD

**Issue by vilius-g** [rollbar/rollbar-php#305]
When reporting fatal errors using rollbar-php, their origin is lost, like this:

```
 E_ERROR: Call to a member function getChoices() on null: Call to a member function getChoices() on null
1
File "/var/sites/knygos.lt/www/vendor/rollbar/rollbar/src/Rollbar.php", line 191 in Rollbar\DataBuilder::generateErrorWrapper
2
File "/var/sites/knygos.lt/www/vendor/rollbar/rollbar/src/Rollbar.php", line 168 in Rollbar\Rollbar::generateErrorWrapper
3
File "/var/sites/knygos.lt/www/vendor/rollbar/rollbar/src/DataBuilder.php", line 1001 in Rollbar\Rollbar::fatalHandler
```

There is no information in the report about the original file and line that caused the error. These are the most important details in cases like this and not including them makes the reporting essentially useless for debugging.

**Comment by rivkahstandig3636** [comment]
Thanks for letting us know! We'll look into it.

**Comment by ArturMoczulski** [comment]
This is resolved in the latest master and will be included in the new `1.4.0` tag.
